src/components/MenuBurger/style.jsx

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+
+const SFormMenuBurger = styled.div`
+  @media only screen and (max-width: 750px) {
+    .bm-burger-button {
+      position: fixed;
+      width: 36px;
+      height: 30px;
+      left: 36px;
+      top: 28px;
+    }
+    .menu {
+      height: 10vh;
+      position: fixed;
+      top: 0;
+      z-index: 1000;
+      margin-bottom: 1vh;
+      width: 100%;
+      background-color: #0081c9;
```

Comment:
I assume this is the blue we use everywhere on the site? If that's the case, use global variable instead of raw hexa code